cli/electron.ts

```diff
@@ -372,7 +372,7 @@ function electronGulpTask(taskName: string): Promise<void> {
     const gulpPath = path.join(pxtElectronPath, "node_modules", ".bin", "gulp");
 
     return nodeutil.spawnAsync({
-        cmd: nodeutil.addCmd(gulpPath),
+        cmd: "gulp.cmd",
```

Comment:
Shifting to global is a bit of a workaround but it seamed like the simplest/fastest solution.